deployments/warp_routes/USDCSTAGE/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "USD Coin STAGE",
+  "symbol": "USDCSTAGE",
+  "description": "Warp Route USDCSTAGE",
+  "image": "https://raw.githubusercontent.com/hyperlane-xyz/hyperlane-registry/mo/usdc-eclipse-staging/deployments/warp_routes/USDCSTAGE/logo.svg",
```

Comment:
I think this is going to break once you merge the pr as it is referencing your branch